src/test/java/org/example/P07_LeapYearTest.java

```diff
@@ -0,0 +1,26 @@
+package org.example;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class P07_LeapYearTest {
+
+    @Test
+    void year_evenly_divisible_by_400_is_leap_year() {
+        int myYear = 2024;
```

Comment:
Put back input `int myYear = 2000;` to match test name `year_evenly_divisible_by_400_is_leap_year`

Reply:
done